cmd/clusterlink/cmd/deploy/deploy_peer.go

```diff
@@ -47,8 +59,9 @@ type PeerOptions struct {
 	Namespace string
 	// CertDir is the directory where the certificates for the fabric and peer are located.
 	CertDir string
-	// StartInstance, if set to true, deploys a ClusterLink instance that will create the ClusterLink components.
-	StartInstance bool
+	// StartInstance, represents which component to deploy:
+	// `all` (clusterlink components and operator), `operator`, or `none`.
+	StartInstance string
```

Comment:
`Components`?
I think `if StartInstance == "StartNone {` might be ne as readable.